CalendarApp/GoogleCalendarService/BookingService.cs

```diff
@@ -48,11 +48,27 @@ public Event PostEvent(Event eventForAdd, string calendarId)
             }
         }
 
-        public IList<Event> GetEvents(string calendarId)
+        public List<Event> GetEvents(string calendarId)
         {
+            int maxResults = Convert.ToInt32(AppSettingsHelper.GetAppSetting(AppSetingsConst.MaxResults));
+
             try
             {
-                return _service.Events.List(calendarId).Execute().Items;
+                var setUpRequset = _service.Events.List(calendarId);
```

Comment:
You have to be convenient in using var's. The common rule it to use it if u have to or if return type is obvious. In this case it is hard to understand what will be returned